uiowa/uiowa
# Feature: Fixes for IE11 and component inconsistencies/bugs

# How to test:

Run `blt frontend` 

**Responsive videos:** Add a youtube video to a text area and select a size for display.  Save the page and verify that video is full-width. 

**CTA Block:**  Add a cta block and remove the title or summary field.  The markup for the empty field should no longer display when the page is saved. 

**Black brain pattern:**  Add a section and select the "brain pattern black" to the section and make sure that the color is correct after saving. 

**Card equal height:** Go to https://uiowa.local.drupal.uiowa.edu/academics/leading-programs and verify that cards are all equal height. 

**4-col responsive updates:** Go to https://uiowa.local.drupal.uiowa.edu/academics/leading-programs and verify that 4 columns collapse to two columns.

**Horizontal nav active style:**  Go to https://uiowa.local.drupal.uiowa.edu/academics/leading-programs and verify that there is an active underline on "Leading Programs."

**Alert region:** Add a custom text block to the Alert region.  Use this text: 
```
<p><span class="fa-exclamation-circle fas"></span> The university is delaying the start of virtual instruction by one week.&nbsp;Courses will now resume Monday, March 30, and will be conducted virtually through the rest of the spring semester.&nbsp;<a href="https://coronavirus.uiowa.edu">Stay current with campus COVID-19 information</a>.</p>
```
**UIDS form style integration:** Go to https://uiowa.local.drupal.uiowa.edu/contact and verify that the form is being styled.  Also, use the top search menu toggle and verify that the search form is styled correct. 

**Search Quick Links Icons:**  Add two links to the 'Search links" menu.   Verify that each link has an icon next to the link. 

**Page Title Breadcrumbs:**  Verify that page title appears below breadcrumbs on https://uiowa.local.drupal.uiowa.edu/research.

**Skip Link:**  Tab through a page (in chrome) and verify that the skip link appears. 

**IE 11:**  Open https://uiowa.local.drupal.uiowa.edu/ in an IE11 emulator and verify that the page looks somewhat decent.
